### DISCUSSION TITLE
DOCS, build commands, and other old stuff

### DISCUSSION THREAD

**Discussion by rcadena** [leewoodridge/RenegadeBBS#5] [General]
Hi,

I was wondering where the DOCS were going to kept and maintained if not in the repo.  I had a fork that was quite old I'd been working on and after updating noticed that the docs directory was gone.  Would you consider bringing it back in?

Regarding building, I was wondering if you were planning on adding some build information for some minimal systems.  Some context: I had been using BUILDBP.CMD with some slight modifications since I was building in DOSbox and with TP7.  I have both my own build batch file and also some TP7 config files.  So, I have both command line build ability and the ability to build inside TP7.  I intend to clean that up over the next few days and put them as a branch on my fork.  But, I wanted to know if there was some way you all had been doing builds.

**Comment by Renegade-Exodus** [comment]
Feel free to add it.   I just did manually on my system.  Never thought of putting them on git.  I also just build inside tp7

Get Outlook for Android<https://aka.ms/AAb9ysg>
________________________________
From: Robert Cadena ***@***.***>
Sent: Thursday, September 12, 2024 7:55:04 PM
To: leewoodridge/RenegadeBBS ***@***.***>
Cc: Subscribed ***@***.***>
Subject: [leewoodridge/RenegadeBBS] DOCS, build commands, and other old stuff (Discussion #5)


Hi,

I was wondering where the DOCS were going to kept and maintained if not in the repo. I had a fork that was quite old I'd been working on and after updating noticed that the docs directory was gone. Would you consider bringing it back in?

Regarding building, I was wondering if you were planning on adding some build information for some minimal systems. Some context: I had been using BUILDBP.CMD with some slight modifications since I was building in DOSbox and with TP7. I have both my own build batch file and also some TP7 config files. So, I have both command line build ability and the ability to build inside TP7. I intend to clean that up over the next few days and put them as a branch on my fork. But, I wanted to know if there was some way you all had been doing builds.

**Reply by rcadena** [reply]
Another question is how do you guys create a full release with all the other files like ANS, DOCS, and so on?

**Comment by Renegade-Exodus** [comment]
manually.

**Reply by rcadena** [reply]
got it. And where are all the docs, lang, and data files currently located? 
I know someone can go to rgbbs.info and download a release and install, but was wondering if someone wanted to build everything from the repo.

**Comment by Renegade-Exodus** [comment]
Not on the site, only in the full install.   Never has anyone ask for the
docs on GIT before.

The only thing on GIT is the source to compile RG, nothing more, nothing
less.   Again, never had anyone ask for this before on GIT, so it's
catching me off guard.

**Reply by rcadena** [reply]
ah, don't mean to pressure or anything. Just getting to know the project.  
The reason I was asking for the docs and all the other stuff is I was thinking about how all the stuff related to it would be tracked and versioned and maybe, in the future if some critical team member doesn't want to or have the time to make releases, how that would be done if it was not in the repo.  Anyway, like I said, no pressure.

**Comment by Renegade-Exodus** [comment]
No, I appreciate the questions.  I just never had anyone ask. :)
